src/mon/FSCommands.cc

```diff
@@ -295,16 +295,17 @@ class SetHandler : public FileSystemCommandHandler
         });
       }
     } else if (var == "balancer") {
-      if (val.empty())
+      if (val.empty()) {
         ss << "unsetting the metadata load balancer";
-      else 
+      } else {
         ss << "setting the metadata load balancer to " << val;
         fsmap.modify_filesystem(
             fs->fscid,
             [val](std::shared_ptr<Filesystem> fs)
         {
           fs->mds_map.set_balancer(val);
         });
+      }
```

Comment:
IMHO, the braces helps with the readability. i guess we should keep them. and as you suggest, we should unindent the `fsmap.modify_filesystem()` statement.